v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/uniformsplitter/transforms/ReadWithUniformPartitions.java

```diff
@@ -339,7 +363,7 @@ private PCollection<ImmutableList<Range>> peekRanges(
   }
 
   public static long inferMaxPartitions(long count) {
-    return Math.max(1, Math.round(Math.floor(Math.sqrt(count) / 10)));
+    return Math.max(1, Math.round(Math.floor(Math.sqrt(count) / 20)));
```

Comment:
take this number - 20 as a constant and justify why it is set to 20 in the constant deescription.

Reply:
`Math.max(1, Math.round(Math.floor(Math.sqrt(count) / 10` we have taken this (verbatim) from beam's JdbcIO.
The splitter generates initial ranges in powers of 2, so thats close to double of this number, making it 20 makes the total number of inital splits similar to jdbcIO.

Reply:
Wrapped it in a constant in latest push.